requirements-app.txt

```diff
@@ -19,63 +19,64 @@ eventlet ~= 0.33.3
 # https://developers.yubico.com/WebAuthn/
 fido2 >= 1.1.2
 
-Flask >= 2.3.3
+Flask >= 3.0.2
 Flask-Bcrypt >= 1.0.1
 Flask-Cors >= 4.0.0
-Flask-JWT-Extended >= 4.5.2
+Flask-JWT-Extended >= 4.6.0
 
 # 19 April 2023: v0.15.0 depends on marshmallow >= 3.0.0.
 flask-marshmallow ~= 0.14.0
 
-Flask-Migrate >= 4.0.4
+Flask-Migrate >= 4.0.5
 Flask-SQLAlchemy >= 3.0.5
 git+https://github.com/department-of-veterans-affairs/notification-utils.git
 
 # The production image defined by Dockerfile uses Gunicorn.  The development image defined by Dockerfile.local
 # runs the application with the Flask development server by default.
-#
-# This is Gunicorn v20.1.0 with additional fixes for import errors that occur with eventlet > 0.30.2.
-gunicorn[eventlet] @ git+https://github.com/benoitc/gunicorn.git@6a8ebb4844b2f28596ffe7421eb9f7d08c8dc4d8
+gunicorn[eventlet] >= 21.2.0
 
-iso8601 >= 2.0.0
-itsdangerous >= 1.1.0
-jsonschema >= 4.19.0
-kombu >= 5.3.1
+iso8601 >= 2.1.0
+itsdangerous >= 2.1.2
+jsonschema >= 4.21.1
+kombu >= 5.3.5
 
 # 31 March 2023: marshmallow 2.x has not been supported since 18 August 2020.
-marshmallow ~= 2.20.5
+marshmallow ~= 2.21.0
 
 marshmallow-sqlalchemy ~= 0.22.3
 nanoid >= 2.0.0
-notifications-python-client >= 8.1.0
+notifications-python-client >= 9.0.0
 
 # Sending an SMS message with v3 endpoints uses this package to test for a valid phone number.  The package
 # is also a dependency of notification-utils.
 phonenumbers ~= 8.12.57
 
-psycopg2-binary >= 2.9.7
+psycopg2-binary >= 2.9.9
 pwnedpasswords >= 2.0.0
 PyJWT >= 2.8.0
 # 4 Dec 2023: Ignoring security vulnerability 62123 in Makefile. It is fixed in >= 3.17
-pypdf >= 3.17.2
+pypdf >= 4.0.2
 
-python-dotenv >= 1.0.0
+python-dotenv >= 1.0.1
 python-magic >= 0.4.27
 
 # jsonschema requires this package to validate the date-time format but doesn't include it as a dependency.
 # Validation fails silently without it.
 # https://python-jsonschema.readthedocs.io/en/latest/validate/?highlight=date-time#validating-formats
 rfc3339-validator >= 0.1.4
 
-sentry-sdk[flask] >= 1.31.0
+sentry-sdk[flask] >= 1.40.5
 
 # 10 December 2022: Ignoring security vulnerability 51668 in Makefile.  It is fixed in  >= 2.0.0b1.
-SQLAlchemy ~= 1.4.49
+SQLAlchemy ~= 1.4.51
 
-twilio >= 8.8.0
-Unidecode >= 1.3.6
+twilio >= 8.13.0
+Unidecode >= 1.3.8
 validatesns >= 0.1.1
 
 # Flask depends on this package.  It's listed here as a top level dependency--intentionally with no version--because
 # it's directly imported in v3 error handling code to reference the exception BadRequest.
 Werkzeug
+
+# eventlet uses this package, but upgrading past 2.4.2 causes DNS errors.
+dnspython == 2.4.2
```

Comment:
Consistent with the rest of the file, please put this in alphabetical order, and date the explanation (i.e. "23 February 2024:").

Reply:
Ok, I've put dnspython in the correct spot